nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/test/java/org/apache/nifi/processors/standard/TestReplaceText.java

```diff
@@ -404,6 +404,27 @@ public void testRegexWithExpressionLanguageIsEscaped() throws IOException {
         out.assertContentEquals("Hello, World!");
     }
 
+    /**
+     * Test for NIFI-5474
```

Comment:
Probably makes sense to explain the actual problem here rather than referencing the JIRA number so that if looking at it later it's easier to understand